src/dune_util/report_error.ml

```diff
@@ -200,6 +200,7 @@ let gen_report exn backtrace =
       | User -> msg
       | Developer -> append msg (i_must_not_crash ())
     in
+    Log.info_user_message msg;
```

Comment:
This needs to be `Log.log`

Reply:
I've also modified the type of `Log.log` to take `User_message.t` rather than `Pp.t list` which makes it more usable.